### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [BarryJin/prj-rev-bwfs-tea-cozy#1]
Each image should have descriptive alt text. This helps the visually impaired through the use of screen readers know what each image is about.
